app/src/main/java/com/example/robin/roomwordsample/fragments/NewToDoFragment.kt

```diff
@@ -40,7 +40,7 @@ class NewToDoFragment : Fragment() {
     var hr = 0
     var min = 0
     private lateinit var wordViewModel: WordViewModel
-    val colors = resources.getStringArray(R.array.colors)
+    private lateinit var colors: Array<String>
```

Comment:
Why declare colors as a lateinit when you already know it's value?
Use val instead and declare it's value here.

Reply:
It can't be initialised outside onCreateView since it requires context to fetch resources. It threw an error, that's why I have used lateinit.

Reply:
Try to initialise using lazy.

Reply:
Done.